src/ppx/ppx_lwt.mli

```diff
@@ -193,6 +193,21 @@ $ ocamlfind ocamlc -package lwt.ppx \
 
    {2 Sequence}
 
+   It is possible to sequence Lwt operations:
+   {[
+     [%lwt write stdout "Hello again, "; write stdout "world!"]
+   ]}
```

Comment:
I wouldn't recommend this syntax. We are trying to get rid of `[%lwt ...]` completely because it is fragile to refactoring the inside. We can't actually ban the syntax, but we shouldn't encourage it IMO.